apps/fluent-tester/src/TestComponents/Badge/CounterBadgeTest.tsx

```diff
@@ -2,17 +2,17 @@
 import React from 'react';
 import { View, Platform, Text } from 'react-native';
 import { CounterBadge } from '@fluentui-react-native/badge';
-import { SvgIconProps } from '@fluentui-react-native/icon';
-import TestSvg from '../../FluentTester/test-data/test.svg';
+import { svgProps } from '../Common/iconExamples';
+import { IconSourcesType } from '@fluentui-react-native/icon';
 
 export const CounterBadgeTest: React.FunctionComponent = () => {
-  const svgProps: SvgIconProps = {
-    src: TestSvg,
-    viewBox: '0 0 500 500',
-  };
-
   const svgIconsEnabled = ['ios', 'macos', 'win32', 'android'].includes(Platform.OS as string);
 
+  const iconProps: IconSourcesType = { svgSource: svgProps };
```

Comment:
can we factor the platform concern to iconExamples as well, since we have different colors and code managing the same thing?  Though I wonder a bit why the color matters at all for android?

Reply:
For android, icons passed into a button / badge don't seem to show unless a color is given. Not really sure why that is either. Seems to be an issue?

Reply:
Hmm that doesn't bode well for making icons show with default colors. @ankraj12 @rohanprasadofficial @ayush547 FYI

Reply:
I'm inclined to agree with this as well - the hook will go so we're not obscuring the bug.